NEWS

```diff
@@ -41,6 +41,7 @@ New classes supported by the `pander` generic S3 method:
  * lrm
  * summary.rms
  * polr/summary.polr
+ * ets
```

Comment:
Please move this into separate category, since technically this change is not in pander 0.6